### PR TITLE
Add support for peek and length on the coordinator and bump version:

### PR DESCRIPTION
**Pull request by Soleone**
- length returns total amount of tasks across all queues
- peek returns the top task from each queue in an array

please review @tylermercier 
fyi @aprofeit @jordanwheeler @boourns

### CODE REVIEW

**Comment by boourns** [comment]
:ship:

**Comment by aprofeit** [comment]
Should we move this repo to /shopify/coordinator ?

**Comment by Soleone** [comment]
maybe @aprofeit, if only so that auto-completing github username works in PRs, i had to copy and paste from a github textbox in a shopify project :P

File: lib/coordinator/base.rb [hunk 1]
```diff
@@ -38,6 +38,14 @@ def info(skill)
       }
     end
 
+    def length
```
Comment (tylermercier) [inline lib/coordinator/base.rb]: Does length make sense? This is the total for all queues.

Comment (Soleone) [inline lib/coordinator/base.rb]: just tried to make it symmetric to the queue class which also uses length.

i thought you can call length either on a single queue or the whole coordinator

File: lib/coordinator/base.rb [hunk 2]
```diff
@@ -38,6 +38,14 @@ def info(skill)
...
+      @queues.inject(0) { |sum, queue| sum + queue.length }
+    end
+
+    def peek
```
Comment (tylermercier) [inline lib/coordinator/base.rb]: Same here. This is all top items from each of the queues.

File: lib/coordinator/base.rb (hunk 1 above)
Comment (Soleone) [inline lib/coordinator/base.rb]: you could potentially mistake it to return the amount of queues instead i guess. but not sure what a better still simple name would be right now

**Comment by tylermercier** [comment]
LGTM

**Comment by Soleone** [comment]
Renamed to `peek_all` and `length_all` to avoid confusion. You think that works or any better suggestion?

**Pull request merged**